src/docs/environment/index.mdx

```diff
@@ -19,7 +19,7 @@ You're going to be working out of this repository for the remainder of the setup
 
 You'll need to first install Xcode cli tools. (If you try running `git` or `make`, you'll get an xcrun error. Run `xcode-select --install` and follow the instructions.
 
-Once you have Xcode installed, install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`.
+Once you have Xcode installed, install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`. This may take a few minutes, and brew isn't great about showing a progress bar, so it may look like the process has hung. Have faith and let it run.
```

Comment:
Oh actually, there's a progress bar with `brew bundle --verbose`. Could you just change to that?

Reply:
Why does that matter? Let's not be nit picky about this kind of thing. If folks want to change behavior of things they can figure it out for themselves. We don't need to explain all of technology in our Sentry-specific docs.

Reply:
I'll use this as a reminder in that I already re-wrote these pages once to remove peoples opinions (and a LOT of exces boilerplate text) about what others should do. Let's keep this to just straight forward "heres how you do something" without adding a ton of extra content for a persons particular flavor of behavior.

Reply:
This is hardly "explaining all of technology." The vast majority of CLI tools provide progress bars for long processes, so when one doesn't, it's possible it's hung. Is there any harm in adding `--verbose`, and making it so someone and can just cut and paste the commands and still see immediately that it's not hung up?

Reply:
The docs should be concise. They should explain the minimal steps needed to get up and working correctly, without adding additional complexity or repeating themselves (unless absolutely needed). This page used to be a mess of both of those problems. My point is we should not be re-adding that kind of flavor text. For example, I've never once had an issue running 'brew install' because it "didn't show a progress bar", so why do we feel we must force this knowledge on people? How's that any different (or better) than putting a big old "hey almost no CLIs show progress bars, so just wait until they're finished"? That's explaining technology, not explaining how to setup Sentry.

Reply:
Aside, I was reviewing the old PR I guess, but my point is still the same when we make changes to docs.

Reply:
I see your point in general, but I also feel like if adding a flag makes the process more seamless for people, it's worth it. Each time I've gone through this process, I've had the, "Wait, is it frozen? Hmmm... Oh, right, damn, brew is terrible about progress bars. Okay, fine, I'll wait." conversation with myself. A blocker? Hardly. But it's a bump, so would be nice, IMHO, to smooth it out for folks.

Reply:
I'm fine with the `--verbose` flag (though I'd argue this is still not needed - again, assume people understand how to computer), my bigger feedback was when there was much more boilerplate.

Reply:
Okay. We can agree to disagree about the value of the flag. 🙂

The rest is gone now, so no issue there.